CHANGELOG.md

```diff
@@ -1,6 +1,286 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.3.0
+-----
+**Date:** 2017-04-..
+
+C++11: Bremsstrahlung, EmZ, Thomas-Fermi, Clang, New Plugins
+
+This is the first release of PIConGPU requiring C++11. We added a
+newly developed current solver (EmZ), support for the generation of
+Bremsstrahlung, Thomas-Fermi Ionization, Laguerre-modes in the
+Gaussian-Beam laser, in-simulation plane for laser initialization,
+new plugins for in situ visualization (ISAAC), a generalized particle
+calorimeter and a GPU resource monitor. Initial support for clang
+(host and device) has been added and our documentation has been
+streamlined to use Sphinx from now on.
+
+### Changes to "0.2.0"
+
+**.param & .unitless file changes:**
+ - use C++11 `constexpr` where possible and update arrays #1799 #1909
+ - use C++11 `using` instead of `typedef`
+ - removed `Config` suffix in file names #1965
+ - `gasConfig` is now `density`
```

Comment:
OPTIONAL:
adding `.param` and `.unitless`would make it clearer that you mean file names